tests/rpm/BUILD

```diff
@@ -13,19 +13,19 @@
 # limitations under the License.
 # -*- coding: utf-8 -*-
 
-load(":toolchain_tests.bzl", "create_toolchain_analysis_tests")
-load("@bazel_skylib//rules:build_test.bzl", "build_test")
+load("analysis_tests.bzl", "analysis_tests")
+load("toolchain_tests.bzl", "create_toolchain_analysis_tests")
 load(
-    "//:mappings.bzl",
+    "//pkg:mappings.bzl",
     "pkg_attributes",
     "pkg_filegroup",
     "pkg_files",
     "pkg_mkdirs",
     "pkg_mklink",
 )
-load("//:rpm.bzl", "pkg_rpm")
+load("//pkg:rpm.bzl", "pkg_rpm")
+load("@bazel_skylib//rules:build_test.bzl", "build_test")
```

Comment:
Should remain on top

Reply:
I moved it, but I tempted to create (and then fix) a buildifier bug about ordering of load statements.  Using `sort` does not work because standard linux sort has decided to be "human readable" rather than by byte.  Some people like external repos first, some like them last.